draft-ietf-sframe-enc.md

```diff
@@ -836,81 +836,127 @@ The authors wish to specially thank Dr. Alex Gouaillard as one of the early
+necessary to transmit a given media stream.  Exactly much overhead will be added
+depend on several factors:
 
-The number of bytes overhead per frame is calculated as the following
+* How many senders are involved in a conference (length of KID)
+* How long the conference has been going on (length of CTR)
+* The ciphersuite in use (length of authentication tag)
+* Whether SFrame is used to encrypt packets, whole frames, or some other unit
 
-~~~
-1 + FrameCounter length + 4
-~~~
+Overall, the overhead rate in kilobits per second can be estimated as:
 
-The constant 1 is the SFrame header byte and 4 bytes for the HBH authentication tag for both audio and video packets.
-
-## Audio
-
-Using three different audio frame durations
+```
+OverheadKbps = (1 + |CTR| + |KID| + |TAG|) * 8 * UnitsPerSecond / 1024
+```
 
-* 20ms (50 packets/s)
-* 40ms (25 packets/s)
-* 100ms (10 packets/s)
+Here the constant value `1` reflect the fixed SFrame header; `|CTR|` and
+`|KID|` reflect the length of those fields; `|TAG|` reflects the cipher
+overhead; and `UnitsPerSecond` reflects the number of SFrame-encrypted units
+sent per second (e.g., packets or frames per second).
 
-Up to 3 bytes frame counter (3.8 days of data for 20ms frame duration) and 4
-bytes fixed MAC length.
+In the remainder of this secton, we compute overhead estimates for a collection
+of common scenarios.
 
-| Counter len| Packets   | Overhead  | Overhead | Overhead  |
-|            |           | bps@20ms  | bps@40ms | bps@100ms |
-|:----------:|:---------:|:---------:|:--------:|:---------:|
-|          1 | 0-255     |      2400 |     1200 |       480 |
-|          2 | 255 - 65K |      2800 |     1400 |       560 |
-|          3 | 65K - 16M |      3200 |     1600 |       640 |
+## Assumptions
 
-## Video
+In the below calculations, we make conservative assumptions about SFrame
+overhead, so that the overhead amounts we compute here are likely to be an upper
+bound on those seen in practice.
 
-The per-stream overhead bits per second as calculated for the following video
-encodings:
+| Field           | Bytes | Explanataion                                      |
+|:----------------|------:|:--------------------------------------------------|
+| Fixed header    | 1     | Fixed                                             |
+| Key ID (KID)    | 2     | >255 senders; or MLS epoch (E=4) and >16 senders  |
+| Counter (CTR)   | 3     | More than 24 hours of media in common cases       |
+| Cipher overhead | 16    | Full GCM tag (longest defined here)               |
 
-* 30fps @ 1000Kbps (4 packets per frame)
-* 30fps @ 512Kbps (2 packets per frame)
-* 15fps @ 200Kbps (2 packets per frame)
-* 7.5fps @ 30Kbps (1 packet per frame)
+In total, then, we assume that each SFrame encryption will add 22 bytes of
+overhead.
 
-Overhead `bps = (Counter length + 1 + 4 ) * 8 * fps`
+We consider two "units" of encryption, frames and packets.  In each scenario, we
```

Comment:
We call them "units", then refer to them as scenarios immediatel afterward. Should "units" be scenarios?
```suggestion
We consider two scenarios of encryption, frames and packets.  In each scenario, we
```

Reply:
Also wondering on this, is the comma correct? I suppose without using the oxford comma it does indeed describe the 2 scenarios, but I wonder if this is instead a good time to use a colon:
```suggestion
We consider two "units" of encryption: frames and packets.  In each scenario, we
```